recorder/src/idl/recorder.proto

```diff
@@ -156,6 +156,7 @@ message CpuSampleWork {
   required uint32 frequency = 1;
   required uint32 max_frames = 2;
   optional uint32 serialization_flush_threshold = 3 [default = 100];
+  optional bool capture_error_bt = 4 [default = true];
```

Comment:
Actually, even though effect is a native-backtrace, from user's PoV, it is a native-backtrace captured iff we couldn't grab a java backtrace (in other words, we had an error in capturing java backtrace, so we ended up capturing the next best thing). Hence the name.